packages/core/src/tools/instrumentMethod.ts

```diff
@@ -125,10 +125,13 @@ export function instrumentSetter<TARGET extends { [key: string]: any }, PROPERTY
     return { stop: noop }
   }
 
+  const stoppedInstrumentation = noop
   let instrumentation = (target: TARGET, value: TARGET[PROPERTY]) => {
     // put hooked setter into event loop to avoid of set latency
     setTimeout(() => {
-      after(target, value)
+      if (instrumentation !== stoppedInstrumentation) {
+        after(target, value)
+      }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could we have a test expressing the purpose of this change?

Reply:
https://github.com/DataDog/browser-sdk/blob/b1733892001f9e2baf0e5ee75b7228f6dc3d81bb/packages/core/src/tools/instrumentMethod.spec.ts#L270